jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -618,11 +618,12 @@ def _dot_product_attention_fwd_batcher(
     *_, S, _, _ = key.shape
   B = math.prod(Bs)
   has_bias, _ = variadic_args
+  original_shape = query.shape
```

Comment:
How about `output_shape` since you only use it for `output`?

Reply:
Done.